### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by djino97**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/376

### CODE REVIEW

**Review by KostinP — APPROVED** [review]

**Review by TIove — APPROVED** [review]

File: test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+            _editDbEmailTemplateText = new DbEmailTemplateText
+            {
+                Id = Guid.NewGuid(),
+                EmailTemplateId = _emailTemplateId,
+                Subject = _editEmailTemplateTextInfo.Subject,
+                Text = _editEmailTemplateTextInfo.Text,
+                Language = _editEmailTemplateTextInfo.Language
+            };
+
+
```
Comment (borovHokage) [inline test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs:108]: delete empty line

File: test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+                .Returns(true);
+
+            _validatorMock
+                .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
+                .Returns(false);
+
+            Assert.Throws<ValidationException>(() => _command.Execute(_newEmailTemplate));
+        }
+
+
```
Comment (borovHokage) [inline test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs:163]: delete empty line

File: test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs (hunk 1 above)
Comment (djino97) [reply test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs]: fixed

File: test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs (hunk 2 above)
Comment (djino97) [reply test/MessageService.Business.UnitTests/Workspace/EditEmailTemplateCommandTests.cs]: fixed

**Review by borovHokage — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

**Pull request merged**